docs/hardhat-network/reference/README.md

```diff
@@ -0,0 +1,407 @@
+# Hardhat Network Reference
+
+## Supported hardforks
+
+- byzantium
+- constantinople
+- petersburg
+- istanbul
+- muirGlacier
+
+## Config
+
+- Default behavior: Hardhat will automatically run in-process when `defaultNetwork` is empty or set to `hardhat`.
+
+### Mining modes
```

Comment:
Is this intended to bu a subsection of Config?

Reply:
Well, this section is about config. I think what confuses me are the following ones, that mostly use the RPC

Reply:
@alcuadrado and I had a call about this, and I took these notes on the intended outcome:
```
medium term, give each product its own PAGE, with its own sidebar.
for short term, re-word "Explanation" to "What is it?", add another page to explain mining modes, and link to it in the sidebar as "Mining Modes", and continue the convention of referring to the Divio content types in a markdown comment in the top-level heading of the page, as already done in the hardhat-network/README.md document.
```
I implemented the short term approach here: https://github.com/nomiclabs/hardhat/pull/1657/commits/3383747fecd278d79aa571eb2f06d7fa63997b98